NEWS

```diff
@@ -1,3 +1,10 @@
+BoutrosLab.plotting.general 7.0.0 2022-02-08
+
+UPDATE
+* Renamed ks.test.critical.value() function to critical.value.ks.test()
+  to resolve naming conflict with S3 generic ks.test() from base R stats
+  package.
+
```

Comment:
Maybe you can add a line of dashes here (as line 8) to be consistent with the format below?

Reply:
Good catch!